src/bionics.cpp

```diff
@@ -85,20 +203,26 @@ bionic_data const& bionic_info(std::string const &id)
 
     debugmsg("bad bionic id");
 
-    static bionic_data const null_value {"bad bionic", false, false, 0, 0, 0, 0, 0, "bad_bionic", false};
+    static bionic_data const null_value {"bad bionic", false, false, 0, 0, 0, 0, 0,
+                                         "bad_bionic", false,
+                                         { num_bp , 1 } };
```

Comment:
I pulled your branch and tried to compile it. Double curly brackets fixed all the compile errors except one.
```
src/bionics.cpp:2342:9: error: no matching constructor for initialization of 'bionic_data'
        bionic_data( std::move( name ), power_source, toggled, on_cost, off_cost,
        ^            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~        

```        
Probably it's caused by incompatibility of `<body_part, int>` and `<body_part, size_t>` (signed `int` and unsigned `int` respectively).